project_setup.sh

```diff
@@ -12,15 +12,19 @@ echo "generating project files..."
 echo 'const std = @import("std");
 const rl = @import("raylib-zig/build.zig");
 
-pub fn build(b: *std.Build) void {
+pub fn build(b: *std.Build) !void {
     const target = b.standardTargetOptions(.{});
     const optimize = b.standardOptimizeOption(.{});
-
-    var raylib = rl.getModule(b);
-    var raylib_math = rl.math.getModule(b);
+    //web exports are completely separate, due to the amount of hackery required.
+    if (target.getOsTag()) {
```

Comment:
This seems like it either wouldn't build or causes unexpected bugs

Reply:
I am working on changing that area of the code to be more flexible and sensible, so don't worry about it for now.